api/v1/kataconfig_types.go

```diff
@@ -35,10 +35,6 @@ type KataConfigSpec struct {
 	// +kubebuilder:default:=false
 	CheckNodeEligibility bool `json:"checkNodeEligibility"`
 
-	// KataMonitorImage is used to specify the container image used for kata-monitor
-	// +kubebuilder:default:="quay.io/openshift_sandboxed_containers/openshift-sandboxed-containers-monitor:latest"
-	KataMonitorImage string `json:"kataMonitorImage"`
-
```

Comment:
Since this modifies the CRD, anything required to handle upgrades from a previously created  CR having this value ?

Reply:
`KataMonitorImage` should also be dropped from the controller test here:

https://github.com/openshift/sandboxed-containers-operator/blob/master/controllers/openshift_controller_test.go#L281

Reply:
@gkurz good catch, thanks!

Reply:
Does anybody know how to address @bpradipt 's concern above? @jensfr perhaps?

Reply:
> Does anybody know how to address @bpradipt 's concern above? @jensfr perhaps?

It seems that the usual process when changing the CRD is to bump the api version and setup conversion webhooks to be able to expose an older object through the new api. This being said, I don't know if we really need all that in this case. If we have OSC 1.3 and a kataconfig already deployed, we want the upgrade to deploy the new monitor images (see KATA-1650) and we thus don't really care for any pre-existing value in the kataconfig.

Reply:
so what happens when you try to create an 'old' kataconfig with monitorImage set after this change to the CRD is merged?